src/components/Main/main.jsx

```diff
@@ -1,6 +1,7 @@
 import Banner from "./Banner/banner";
```

Comment:
The component is almost identical to we have already created. Please, make work on your own and explore various ways to implement